### ISSUE TITLE
At startup chrome open 2 about:blank tabs 

### ISSUE THREAD

**Issue by LeMoussel** [GoogleChrome/chrome-launcher#161]
**Windows**: 10 x64
**Node**: v10.15.0
**Chrome**: 75.0.3770.142

When lauching llike this:
```
  // https://github.com/GoogleChrome/chrome-launcher#launch-options
  const opts = {
    port: 1111,
    logLevel: 'info',
    // logLevel: 'silent',
    output: 'json',
    disableDeviceEmulation: true,
    defaultViewport: {
      width: 1200,
      height: 900,
    },
    chromeFlags: [
      // https://github.com/GoogleChrome/chrome-launcher/blob/master/docs/chrome-flags-for-tools.md
      // '--headless',
      '--disable-mobile-emulation',
      '--no-sandbox',
      '--show-paint-rects',
      '--disable-extensions',
      '--disable-background-networking',
      '--disable-sync',
      '--disable-default-apps',
      '--mute-audio',
      '--no-first-run',
      '--enable-automation',
    ],
  };

  const chromeBrowser = await chromeLauncher.launch(opts);

```
I get the following:
```
ChromeLauncher No debugging port found on port 1111, launching a new Chrome. +0ms
ChromeLauncher Waiting for browser. +12ms
ChromeLauncher Waiting for browser... +3ms
ChromeLauncher Waiting for browser...√ +503ms
```
And a chrome window opens with two `about:blank` tabs

![image](https://user-images.githubusercontent.com/1816573/61356125-f8c7e280-a875-11e9-8506-b21ffce0899b.png)

**Comment by patrickhulce** [comment]
The default `startingUrl` option is `about:blank`, if you'd like it to start on a different URL, pass a different URL as `startingUrl`.

By the way, it looks like you might be confusing some flags, `output`, `disableDeviceEmulation`, and `defaultViewport` are not `chrome-launcher` flags. Perhaps you were thinking of [lighthouse](https://github.com/GoogleChrome/lighthouse) for some of those?

**Comment by LeMoussel** [comment]
I use `lighthouse` in  my code :)
If I set `startingUrl` option with an Url (eg. example.com), I have two tabs with with example.com site page..

**Comment by patrickhulce** [comment]
I'm not able to reproduce this on Mac. Is the double tab behavior what you see if you run Chrome yourself from the command prompt without the flags?

```
<path to chrome.exe> --no-first-run --remote-debugging-port=1111 --user-data-dir=C:\Windows\Temp\test1234  http://example.com
```

And with the flags?
```
--disable-translate --disable-extensions --disable-background-networking --safebrowsing-disable-auto-update --disable-sync --metrics-recording-only --disable-default-apps --mute-audio
```

If so, then it seems like that's Chrome's behavior in your environment. You could file it at http://crbug.com if it seems like a bug to you.

**Comment by patrickhulce** [comment]
Also just to double check you're not passing the URL in flags *and then also* setting `startingUrl` are you? It would be expected to open 2 tabs then.

**Comment by LeMoussel** [comment]
_**Run Chrome from the command prompt without the flags :**_
> "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --no-first-run --remote-debugging-port=1111 --user-data-dir=C:\Windows\Temp\test1234  http://example.com


![image](https://user-images.githubusercontent.com/1816573/61432830-e27f5c80-a931-11e9-945c-ba890b97612b.png)

_**Run Chrome from the command prompt flags :**_
> 
"C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --disable-translate --disable-extensions --disable-background-networking --safebrowsing-disable-auto-update --disable-sync --metrics-recording-only --disable-default-apps --mute-audio http://example.com

![image](https://user-images.githubusercontent.com/1816573/61433171-df38a080-a932-11e9-8113-ed1f1e7fc464.png)

**Results:** Only one tab in both cases

**Comment by LeMoussel** [comment]
Here is my test code.  URL is not passing in flags and then also not setting in `startingUrl`.
URL is setting in  lighthouse options.

```
// https://github.com/GoogleChrome/chrome-launcher
const chromeLauncher = require('chrome-launcher');
// https://github.com/GoogleChrome/lighthouse
const lighthouse = require('lighthouse');
const lighthouseConfig = require('lighthouse/lighthouse-core/config/lr-desktop-config.js');

(async () => {
  // https://github.com/GoogleChrome/chrome-launcher#launch-options
  const opts = {
    logLevel: 'info',
    // logLevel: 'silent',
    output: 'json',
    disableDeviceEmulation: true,
    defaultViewport: {
      width: 1200,
      height: 900,
    },
    chromeFlags: [
      // https://github.com/GoogleChrome/chrome-launcher/blob/master/docs/chrome-flags-for-tools.md
      // '--headless',
      '--disable-mobile-emulation',
      '--no-sandbox',
      // '--show-paint-rects',
      '--disable-extensions',
      '--disable-background-networking',
      '--disable-sync',
      '--disable-default-apps',
      '--mute-audio',
      '--no-first-run',
      '--enable-automation',
      '--disable-gpu',
    ],
  };

  // Open 2 two about:blank tabs
  const chromeBrowser = await chromeLauncher.launch(opts);
  // opens a 3rd tab with exemple.com
  const lighthouseReport = await lighthouse(
    'http://example.com',
    {
      port: chromeBrowser.port,
      disableDeviceEmulation: true,
    },
    lighthouseConfig,
  ); 

  console.log(`Chrome debugging port running on ${chromeBrowser.port}`);

  chromeBrowser.kill();
})().catch(e => {
  console.error(e);
  process.exit(1);
});
```

**Comment by patrickhulce** [comment]
Sorry one last debug step for you to try then :)

Turn on `verbose` logging for ChromeLauncher and run the command exactly as it prints in out to you. One of the missing flags must be resulting in the extra tab.

**Comment by LeMoussel** [comment]
With `verbose ` logging for ChromeLauncher

>   ChromeLauncher:verbose created D:\Temp\AppData\Local\Temp\lighthouse.92997946 +0ms
>   ChromeLauncher:verbose Launching with command:
>   ChromeLauncher:verbose "C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --disable-translate --disable-extensions --disable-background-networking --disable-sync --metrics-recording-only --disable-default-apps --mute-audio --no-first-run --remote-debugging-port=64753 --user-data-dir=D:\Temp\AppData\Local\Temp\lighthouse.92997946 --disable-mobile-emulation --no-sandbox --disable-extensions --disable-background-networking --disable-sync --disable-default-apps --mute-audio --no-first-run --enable-automation --disable-gpu about:blank +16ms
>   ChromeLauncher:verbose Chrome running with pid 772 on port 64753. +0ms
>   ChromeLauncher Waiting for browser. +0ms
>   ChromeLauncher Waiting for browser... +0ms
>   ChromeLauncher Waiting for browser...√ +528ms
> Chrome debugging port running on 64753
>   ChromeLauncher Killing Chrome instance 772 +5s

![image](https://user-images.githubusercontent.com/1816573/61475670-9d881400-a98b-11e9-887e-4c379260dba5.png)

**Comment by patrickhulce** [comment]
Right so does running 

```
C:\Program Files (x86)\Google\Chrome\Application\chrome.exe" --disable-translate --disable-extensions --disable-background-networking --disable-sync --metrics-recording-only --disable-default-apps --mute-audio --no-first-run --remote-debugging-port=64753 --user-data-dir=D:\Temp\AppData\Local\Temp\lighthouse.92997946 --disable-mobile-emulation --no-sandbox --disable-extensions --disable-background-networking --disable-sync --disable-default-apps --mute-audio --no-first-run --enable-automation --disable-gpu about:blank
```

yield the same result? If it does, one of the flags here that you didn't run earlier must be responsible for the extra tab.

**Comment by LeMoussel** [comment]
No. One `about:blank` tab.

![image](https://user-images.githubusercontent.com/1816573/61512087-24bfa100-a9f9-11e9-9fa5-86aec25c20c6.png)

**Comment by bbert** [comment]
Same issue for me, on Windows.
I always got 2 tabs opened when using chrome-launcher.
Settings the flags you suggested in chrome-launcher options does not change anything.
Running the same command line as logged by chrome-launcher in a prompt opens a chrome with only one tab. But using chrome-launcher, there is always 2 tabs opened.

**Comment by patrickhulce** [comment]
Wild! Thanks everyone!

It must be a Windows-specific chrome-launcher bug then. @wardpeet any chance you've bumped into this before and had a workaround? :)

**Comment by mattwilliams** [comment]
Mac OS 10.13.6,  chrome-launcher: 0.10.7  chrome: Version 70.0.3538.67 (Official Build) (64-bit)
Same problem.

**Comment by patrickhulce** [comment]
Are you 100% the version is correct @mattwilliams? I meant to come back the other day, but it just became reproducible for me on Mac with Tuesday's Chrome Canary, I'm not able to observe this behavior on Mac on Chrome stable.

**Comment by patrickhulce** [comment]
We believe this to be fixed in 0.11.2 by https://github.com/GoogleChrome/chrome-launcher/pull/162.

Thanks all!

**Comment by LeMoussel** [comment]
It's OK with version: 0.11.2 👍
